### ISSUE TITLE
Please Enchance Map 255 Vertex Limit

### ISSUE THREAD

**Issue by kadavrahaxball** [haxball/haxball-issues#1848]
If the map has over than 255 vertexes game doesnt allow to load this map. Haxball map making industry is improved so much. If the vertex limit enchance it will be better for us. For example i make a soccer pitch but i cannot add stands and crowd or stadium.

**Comment by ghost** [comment]
#1346 Also it can contain this feature

**Comment by kadavrahaxball** [comment]
> #1346 Also it can contain this feature

yes that would be very nice

**Comment by ChasmSolacer** [comment]
Agreed. There could be a new decoration category or something that doesn't need to have collisions or physics to prevent lags
